project/plugins.sbt

```diff
@@ -3,25 +3,15 @@ resolvers += Resolver.url("HMRC-open-artefacts-ivy2", url("https://open.artefact
   Resolver.ivyStylePatterns
 )
 
-sys.env.get("PLAY_VERSION") match {
-  case Some("2.8") => // required since we're cross building for Play 2.8 which isn't compatible with sbt 1.9
-    libraryDependencySchemes += "org.scala-lang.modules" %% "scala-xml" % VersionScheme.Always
-  case _           => libraryDependencySchemes := libraryDependencySchemes.value // or any empty DslEntry
-}
-
-sys.env.get("PLAY_VERSION") match {
-  case Some("2.8") => addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.20")
-  case Some("2.9") => addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.9.0")
-  case _           => addSbtPlugin("org.playframework" % "sbt-plugin" % "3.0.0")
-}
+addSbtPlugin("org.playframework" % "sbt-plugin" % "3.0.0")
 
 addSbtPlugin("org.irundaia.sbt" % "sbt-sassify" % "1.4.11")
 
 addSbtPlugin("org.scoverage" % "sbt-scoverage" % "2.0.9")
 
 addSbtPlugin("net.ground5hark.sbt" % "sbt-concat" % "0.2.0")
 
-addSbtPlugin("com.typesafe.sbt" % "sbt-uglify" % "2.0.0")
+addSbtPlugin("com.github.sbt" % "sbt-uglify" % "3.0.0")
```

Comment:
moved at some point https://github.com/sbt/sbt-uglify

Reply:
@oscarduignan There's a bobby warning about this version of `sbt-sasify`:
```
[0m[[0m[33mwarn[0m] [0m[0mWARNING: Your build has 1 bobby warning(s). Please take action to fix these before the listed date, or they will become violations that fail your build[0m
[0m[[0m[33mwarn[0m] [0m[0m (1) org.irundaia.sbt:sbt-sassify (1.4.11)[0m
[0m[[0m[33mwarn[0m] [0m[0m     Reason: To provide support for Apple Silicon move to io.github.irundaia 1.5.2 or higher
```

Reply:
ah I'll bump this thank you!

Reply:
done